src/wallet/rpc/wallet.cpp

```diff
@@ -621,11 +642,8 @@ RPCHelpMan simulaterawtransaction()
     std::shared_ptr<const CWallet> const wallet = GetWalletForJSONRPCRequest(request);
     if (!wallet) return NullUniValue;
     const CWallet* pwallet = wallet.get();
-
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

nit: Unrelated whitespace change